src/screens/sidebar/Sidebar.jsx

```diff
@@ -19,38 +26,20 @@ export const Sidebar = () => {
         </div>
         <div className='sidebar-nav-content'>
           <div className='sidbar-nav-items'>
```

Comment:
suggestion: you could use "ul" element here and wrap ur links in  "li" , and use nav to wrap your elements

Reply:
will change it